docs/installation_guide.md

```diff
@@ -302,110 +348,118 @@ $ sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.notary.yml -f
     ports:
-      - 8888:80
-      - 443:443
+    - 8888:80
+    - 443:443
     depends_on:
-      - postgresql
-      - registry
-      - core
-      - portal
-      - log
+    - postgresql
+    - registry
+    - core
+    - portal
+    - log
     logging:
-      driver: "syslog"
-      options:  
+        driver: "syslog"
+        options:
         syslog-address: "tcp://127.0.0.1:1514"
         tag: "proxy"
-```
+    ```
 
-2.Modify harbor.cfg, add the port to the parameter "hostname"  
+2.Modify `harbor.cfg`, adding the port to the `hostname` parameter:
 
-```  
-hostname = 192.168.0.2:8888
-```
+    ```conf
+    hostname = 192.168.0.2:8888
+    ```
+
+3. Re-deploy Harbor using the instructions in the [Managing Harbor's Lifecycle](#managing-harbors-lifecycle) section.
 
-3.Re-deploy Harbor referring to previous section "Managing Harbor's lifecycle".
 ### For HTTPS protocol
-1.Enable HTTPS in Harbor by following this [guide](https://github.com/goharbor/harbor/blob/master/docs/configure_https.md).  
-2.Modify docker-compose.yml  
-Replace the first "443" to a customized port, e.g. 8888:443.  
 
-```
-proxy:
+1.Enable HTTPS in Harbor by following the [Configuring Harbor with HTTPS Access](https://github.com/goharbor/harbor/blob/master/docs/configure_https.md) guide.
+
+2. Modify `docker-compose.yml`, replacing the `443` with a customized port, e.g. `8888:443`.
+
+    ```yaml
+    proxy:
     image: goharbor/nginx-photon:v1.6.0
     container_name: nginx
     restart: always
     volumes:
-      - ./common/config/nginx:/etc/nginx:z
+    - ./common/config/nginx:/etc/nginx:z
     ports:
-      - 80:80
-      - 8888:443
+    - 80:80
+    - 8888:443
     depends_on:
-      - postgresql
-      - registry
-      - core
-      - portal
-      - log
+    - postgresql
+    - registry
+    - core
+    - portal
+    - log
     logging:
-      driver: "syslog"
-      options:  
+        driver: "syslog"
+        options:  
         syslog-address: "tcp://127.0.0.1:1514"
         tag: "proxy"
-```
+    ```
 
-3.Modify harbor.cfg, add the port to the parameter "hostname"  
+3. Modify `harbor.cfg`, adding the port to the `hostname` parameter:
 
-```  
-hostname = 192.168.0.2:8888
-```
-
-4.Re-deploy Harbor referring to previous section "Managing Harbor's lifecycle".
+    ```conf
+    hostname = 192.168.0.2:8888
+    ```
 
+4. Re-deploy Harbor using the instructions in the [Managing Harbor's Lifecycle](#managing-harbors-lifecycle) section.
 
 ## Performance tuning
-By default, Harbor limits the CPU usage of Clair container to 150000 and avoids its using up all the CPU resources. This is defined in the docker-compose.clair.yml file. You can modify it based on your hardware configuration.
+
+By default, Harbor limits the CPU usage of the Clair container to 150000 and prevents it from using excessive CPU resources. This is defined in the `docker-compose.clair.yml` file. You can modify it based on your hardware configuration.
```

Comment:
Can we put some units on this number to make things more clear? Or at least explain what `150000` means in this context?